src/Parser/SQLParser.php

```diff
@@ -268,13 +270,13 @@ private static function buildTokenListFromLexemes(vec<string> $tokens): token_li
         );
       } else if ($token_upper === 'TRUE') {
         $out[] = shape(
-          'type' => TokenType::NUMERIC_CONSTANT,
+          'type' => TokenType::BOOLEAN_CONSTANT,
           'value' => '1',
```

Comment:
should these values be updated to true/false?

Reply:
we could do that, but it's handled fine here - https://github.com/slackhq/hack-sql-fake/pull/55/files#diff-607397c4446b8000f684bf522a8a0e6561187a5fd7a6ffe3a211acfcbcbf33b9R28

let me know if you'd prefer we store the 'TRUE', 'FALSE' here then convert to a boolean in https://github.com/slackhq/hack-sql-fake/pull/55/files#diff-607397c4446b8000f684bf522a8a0e6561187a5fd7a6ffe3a211acfcbcbf33b9R28 using `$token['value'] === 'TRUE' ? true : false`